gf2.cpp

```diff
@@ -842,7 +850,11 @@ void DebuggerGetBreakpoints() {
 
 			for (int i = 0; i < breakpoints.Length(); i++) {
 				if (strcmp(breakpoints[i].fileFull, breakpoint.fileFull) == 0 &&
+						strcmp(breakpoints[i].condition, breakpoint.condition) == 0 &&
 						breakpoints[i].line == breakpoint.line) {
+					char buffer[1024];
+					StringFormat(buffer, 1024, "delete %d", breakpoint.number);
+					DebuggerSend(buffer, true, false);
```

Comment:
I think this `DebuggerSend` should be synchronous, i.e. the last parameter set to `true`. Otherwise `DebuggerGetBreakpoints` will be called multiple times, I think?